examples/constant-force-optical-trap/force-bias-optical-trap.py

```diff
@@ -224,7 +224,7 @@ def main():
     # Initialize MBAR.
     print("Running MBAR...")
     # TODO: change to u_kn inputs
-    mbar = pymbar.MBAR(u_kln, N_k, verbose=True, relative_tolerance=1.0e-10)
+    mbar = pymbar.MBAR(u_kln, N_k, verbose=True, relative_tolerance=1.0e-10, solver_protocol = 'robust')
```

Comment:
Shouldn't `robust` be default?

Reply:
That's a good question.  I think this is to get the framework right for now, and the precise solvers used will need a bit more experimentation - what is "robust", and what is "default".  I'm totally find with using what we think is the most robust for the default, and then having a "faster" option.  I will set those solvers in a final PR.